src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  padding: 0;
+}
+
+.search-bar {
+  display: flex;
+  justify-content: center;
+  border-radius: 4px;
+  width: 100%;
+}
+
+.search-bar_big {
+  margin: 20px 0;
+  height: 70px;
+}
+
+.search-bar_small {
+  height: 42px;
+}
```

Comment:
Create one class (`search-bar`), write all styles there and then use `search-bar_small` modifier to overlap sizes

Big input - is a default state of element (all styles written to one class)
Small input - modified element (sizes are overlapped by modifier class)